keras/02-the-layer-class.md

```diff
@@ -0,0 +1,317 @@
+## The `Layer` class
+
+([Source](https://github.com/keras-team/keras/blob/r2.6/keras/engine/base_layer.py#L84))
+
+The `Layer` class is easy to understand. It is the base class for the neural
+network layers in Keras.
```

Comment:
To give user an overall picture, we might want to state that the addition from layer to module is that layer defines the forward path computation.